src/views/thread/container.js

```diff
@@ -325,9 +325,10 @@ class ThreadContainer extends React.Component<Props, State> {
         isCommunityOwner ||
         isChannelModerator ||
         isCommunityModerator;
-      const isParticipantOrAuthor =
```

Comment:
We do actually use the `isParticipantOrAuthor` var further down below (L518) so we can't just remove it—that's why CI fails.

Reply:
Have an in-progress refactor locally, didn't mean to leave this PR marked as in-review :) thanks!